router/core/src/main/scala/io/buoyant/router/ctx.scala

```diff
@@ -0,0 +1,55 @@
+package io.buoyant.router
+
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.context.Contexts
+import com.twitter.util.Local
+
+package object ctx {
```

Comment:
well, i wanted this to be a package rather than an object, but i was being lazy and wanted to put it in one file without creating a directory...  hence package object.

Moving it into its own directory doesn't really address your concern, though